.gitlab-ci.yml

```
@@ -794,7 +794,8 @@ build-push-windows-multiarch-image:
       if ($env:CI_COMMIT_BRANCH -eq "main" -or $env:CI_COMMIT_TAG -match '^v\d+\.\d+\.\d+$') {
         # only push latest tag for main and stable releases.
         echo "Tagging and pushing ${IMAGE_NAME}:latest"
-        docker manifest create ${IMAGE_NAME}:latest --amend ${IMAGE_NAME}:latest-2019 --amend ${IMAGE_NAME}:latest-2022
+        docker manifest rm ${IMAGE_NAME}:latest
+        docker manifest create ${IMAGE_NAME}:latest ${IMAGE_NAME}:latest-2019 --amend ${IMAGE_NAME}:latest-2022
```

Do we want to remove both amends?
```suggestion
        docker manifest create ${IMAGE_NAME}:latest ${IMAGE_NAME}:latest-2019 ${IMAGE_NAME}:latest-2022
```

---

I would suggest removing the --amend since we create a new manifest here.